src/main/scala/cromwell/engine/workflow/CallMetadataBuilder.scala

```diff
@@ -37,8 +37,8 @@ object CallMetadataBuilder {
     def extract(job: Any): BackendValues = {
       job match {
         case ji: LocalJob => BackendValues("Local")
-        case ji: JesJob => BackendValues("JES", jobId = Option(ji.jesId.toString), status = ji.jesStatus)
```

Comment:
Oh! gotcha, that makes more sense.

I have plenty of experience seeing Some(theRealString) in Zamboni messages ;)